### ISSUE TITLE
CI: tests requiring ripemd160 fail on GitHub actions

### ISSUE THREAD

**Issue by stuckatsixpm** [stuckatsixpm/antelopy#13]
While they work locally, `aioeos` and `eospy` tests are currently disabled in the GH actions workflow, as actions uses Ubuntu 22 w/ openssl <3.0.7. This has `rmd160` disabled by default, and I can't for the life of me figure out how to get it enabled again.

Tried (with sudo):
* using apt to update
* using `sed` to enable legacy_sect in `/etc/ssl/openssl.conf`  (permission denied) ([StackOverflow](https://stackoverflow.com/questions/74545780/how-to-enable-the-openssl-3-0-legacy-provider-github-actions))
* using an additional `openssl.conf` and pointing the `OPENSSL_CONF` env variable to it ([StackOverflow](https://stackoverflow.com/questions/74545780/how-to-enable-the-openssl-3-0-legacy-provider-github-actions))
* building openssl 3.2.1 from source, then editing that config file.

I'll keep trying to sort out a viable workaround, but the tests can be run locally with `poetry run tox -e eospy, aioeos`

**Comment by callmenemo491** [comment]
hi @stuckatsixpm , here is how i had to configure/enable it on my Ubuntu machine (not in github actions)
1. find out the directory:
```
# openssl version -d
OPENSSLDIR: "/usr/lib/ssl"
```

2. edit file
```
# sudo vi /usr/lib/ssl/openssl.cnf
```

3. make the following changes/additions:
```
openssl_conf = openssl_init

[openssl_init]
providers = provider_sect

[provider_sect]
default = default_sect
legacy = legacy_sect

[default_sect]
activate = 1

[legacy_sect]
activate = 1
```

**Comment by callmenemo491** [comment]
i believe the issue was you are editing/trying to edit the wrong file.. which openssl  is not parsing.

**Comment by stuckatsixpm** [comment]
@callmenemo491 yep, this is something that was already attempted (bullet points 2 and 3 in the original post)

Unfortunately it might have to wait until a newer version GH's ubuntu runners, because newer versions of OpenSSL do support ripemd160
